js/basic_js.js

```diff
@@ -0,0 +1,58 @@
+
+function getSalary(salary) {
+    salary==Number
+    if (n==15) console.log('thuế', 0.3*n*10**6, 'lương ròng', (15-0.3*n)*10**6 )  
+    else if (n==7) console.log('thuế', 0.2*n*10**6, 'lương ròng', (7-0.2*n)*10**6) 
+    else if(n>7 && n<15) console.log('thuế', 0.1*n*10**6, 'lương ròng', (n-0.1*n)*10**6)  
+}
+function getAge(age) {
+    age==Number
+    if (n < 16)
+        console.log('hoc sinh khong du dieu kien')
+    else console.log('hoc sinh du dieu kien')
+}
+function checkNumber(n) {
+    n==Number
+    if (n==100) console.log('số bạn vừa nhập là 100')
```

Comment:
```suggestion
    if (n === 100) console.log('số bạn vừa nhập là 100')
```
Logic hãy viết theo 1 chuỗi có thứ tự để không bị thiếu trường hợp
Khi so sánh thì dùng 3 dấu bằng để chặt chẽ hơn